types/react-loadable/index.d.ts

```diff
@@ -15,7 +16,7 @@ declare namespace LoadableExport {
         isLoading: boolean;
         pastDelay: boolean;
         timedOut: boolean;
-        error: any;
+        error: Error | false;
```

Comment:
This unfortunately is actually `any` because it's possible to `throw` anything. It's the same reason why typescript doesn't allow you to annotate the type of the catch; `try {} catch (e: Error) {}` is a typescript semantic error.

Reply:
ahhh... you're right, thanks for pointing out.